Applications/TransferLearning/import_pb_to_tensorboard.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
use SPDX?

Reply:
This is not our code. This is from tensorflow. So.. I guess It might be better we add or change for the unnecessary cases.